automation/project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ object Dependencies {
 
   val workbenchGoogleV = "0.21-ae11b9f"
   val workbenchGoogle2V = "0.23-7ddf186"
-  val workbenchServiceTestV = "0.21-bdcdd45"
+  val workbenchServiceTestV = "1.0-d4fe49a1-SNAP"
```

Comment:
just checking -- will this be updated to an official version before merging?

Reply:
Yep